PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -54,4 +55,18 @@ public static void main(String[] args) {
         System.out.println("Your twitter feed is full");
     }
     
+    public static String getCurrentTimeStamp(){
```

Comment:
Why did you use this specific Date format? Side note: The format is great and great work! ;)